### ISSUE TITLE
How to setup rules & route for direct domain connection

### ISSUE THREAD

**Issue by digicenter** [XrayR-project/XrayR#358]
Hello, I want to make some configurations where the domains ending with '.ir' and '.tr' can be accessed directly without going through a proxy. I want to perform this task from the server side. Is there anyone who can guide me?

**Comment by Cp0204** [comment]
Please read the xray document

**Comment by wengguanjie1997** [comment]
我想咨询下。如何设置direct标识，我发现这样设置后，流量一样走的是VPS。是否需要增加一个proxy标识，让国外流量走proxy。但是我的vmess配置信息是从v2board拉取生成。这个我该如何取定义呢？
{
        "tag": "direct",
        "protocol": "freedom",
        "settings": {}
    },

**Comment by unknownqp** [comment]
I'm in the same situation, and I'm using V2board with XrayR as well. I'm not exactly sure how and where to enter these codes to make it work correctly.

**Comment by wengguanjie1997** [comment]
Yeah. If I config vmess configuration in the custom_outbound.json file, that my remote configuration of v2board node info is invalid.
So, how to config this part of proxy configuration ? My purpose is to divert traffic on the server. Because I found that turning on the global proxy on the client will cause some domestic traffic to go through Vps。

**Comment by thank243** [comment]
Once the traffic send to server, this traffic will proxy by the server. Your pupose is handle the traffic on client side, So you should goto panel group to deal with the rule from your panel.
